lib/entities/entity.ts

```diff
@@ -67,7 +67,7 @@ class Entity {
    */
   constructor(
     id: string,
-    name: string | Map<string, string> | Record<string, string>,
+    name: string | Map<string, string> | Record<string, string> | undefined | null,
```

Comment:
Same issue as above: undefined and null are not allowed.

Original JSDoc:
```
@param {string|Map} name The human-readable name of the entity.
       Either a string, which will be mapped to english, or a Map / Object containing multiple language strings.
```

Note: Object input is mentioned in the desription but missing in the definition :-)